src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -20,30 +25,33 @@
     @Autowired
     CartMapper mapper;
 
+    @Autowired
+    OrderMapper orderMapper;
+
 
-    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE,produces = MediaType.APPLICATION_JSON_VALUE)
-    public Cart createCart(CartDto cartDto){
-        return service.createCart(mapper.translateToCart(cartDto));
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public CartDto createCart(CartDto cartDto){
+        return mapper.mapToCartDto(service.createCart(mapper.mapToCart(cartDto)));
```

Comment:
Wygląda lepiej, a teraz już konkretnie. Jak spojrzysz do encji Cart to jest tam tylko id, Items i closed (to pole order 1:1 lepiej wykasować, bo jest założona od strony Order i tu nas nie interesuje). W momencie zakładania nowego koszyka nie znamy id (bo go nie ma), items=domyślnie pusta List<>, a closed domyslnie=false, czyli do założenia potrzebujemy ... nic (bo wszystko jest domyślne). Od strony Postmana przesyłamy puste body: {}. Więc nic nie trzeba mapować do serwisu, wystarczy tu dać 
public CartDto createCart() {
            return mapper.mapToDto(service.createCart())
}
a w serwisie:
    public Cart createCart() {
        return cartRepository.save(new Cart());
    }
Trzeba jednak zmodyfikować mappery, bo na razie Tworzysz je w sposób domyślny jako return new cartDto(cart.getId()) i oprócz pola id, reszta jest domyślna, a nie pochodząca z cart